packages/now-build-utils/src/fs/node-version.ts

```diff
@@ -42,15 +48,40 @@ export async function getSupportedNodeVersion(
         );
       }
     } else {
-      if (!silent) {
-        throw new Error(
-          'Found `engines` in `package.json` with an unsupported node range: ' +
-            engineRange +
-            '\nPlease use one of the following supported ranges: ' +
-            JSON.stringify(supportedOptions.map(o => o.range))
-        );
-      }
+      throw new Error(
+        'Found `engines` in `package.json` with an unsupported Node.js version range: ' +
+          engineRange +
+          '\nPlease use one of the following supported ranges: ' +
+          JSON.stringify(supportedOptions.map(o => o.range)) +
+          '\nThis change is a result of a decision made by an upstream infrastructure provider (AWS).' +
```

Comment:
Sorry for being nit-picky, but this reads better to me:

```suggestion
          '\nThis change is the result of a decision made by an upstream infrastructure provider (AWS).' +
```

Reply:
Fixed 👍